lib/evss/gi_bill_status/service.rb

```diff
@@ -12,6 +12,11 @@ def get_gi_bill_status
       rescue Faraday::ParsingError => e
         log_message_to_sentry(e.message, :error, extra_context: { url: BASE_URL })
         EVSS::GiBillStatus::GiBillStatusResponse.new(403)
+      rescue Faraday::TimeoutError
```

Comment:
Looks like this will drop our test coverage below 90%, please add a spec for this and letters.